src/errors.rs

```diff
@@ -13,6 +13,7 @@ pub enum SlackClientError {
     EndOfStream(SlackClientEndOfStreamError),
     SystemError(SlackClientSystemError),
     ProtocolError(SlackClientProtocolError),
+    UrlProtocolError(SlackClientUrlProtocolError),
```

Comment:
Nit: When you look at the name without knowing this is actually Form URL encoding it may confuse people. The naming is a bit too generic. Something like `FormUrlEncodingError`?